hseling-web-cat-and-kittens/hseling_web_cat_and_kittens/templates/base.html

```diff
@@ -0,0 +1,69 @@
+<!doctype html>
+
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <link rel='shortcut icon' type='image/x-icon' href="{{ url_for('static', filename='img/favicon.ico') }}" />  
+
+    <title>{% block title %}CAT&Kittens{% endblock %}</title>
+
+    {% block links %}{% endblock %}
```

Comment:
May be useful to include common links (e.g., for bootstrap, jquery) here (or may be divide into two blocks - "links_common" and "links")

Reply:
FIXME